pkg/cluster/describe.go

```diff
@@ -294,6 +296,14 @@ func PrintClusterDescription(connection *sdk.Connection, cluster *cmv1.Cluster)
 	return nil
 }
 
+func printNodeInfo(replicasInfo string, securityGroups []string) string {
```

Comment:
just nit pick what I just noticed, the function does not print but formats so might be better name `formatNodeInfo`

Reply:
I agree, but it's the current standard for all other functions that are formatting is also to print :laughing: